CharacterCreation/03_Feats.txt

```diff
@@ -124,8 +124,8 @@ Prerequisites:
 *  None
 
 	Make a stealth check to hide your chosen pistol, shotgun, or SMG on your 
-person. To succeed, you need a minimum score of 20. Roll determines DC for 
-enemy to actively search and find your hidden weapon.
+person. To succeed, you must pass a DC 19 check. Total roll determines DC for an

```

Comment:
Aren't skill checks success on tie? DC not-divisible-by-5 is very unusual.

Reply:
To quote basic rules: "Anytime you 
need to Pass a Skill check, there will be a DC (Difficulty Check) you have to 
exceed in order to successfully pass the check."
I'll happily change it back to 20.

Reply:
Nope, you're correct. DC 19 is the equivalent and more style-consistent phrasing. Disregard.